src/backend/src/services/recruitment.services.ts

```diff
@@ -110,6 +110,23 @@ export default class RecruitmentServices {
     return allMilestones;
   }
 
+  /**
+   * Gets all FAQs for the given organization Id
+   * @param organizationId organization Id of the faq
+   * @returns all the faqs from the given organization
+   */
+  static async getAllFaqs(organizationId: string) {
+    const allFaqs = await prisma.frequentlyAskedQuestion.findMany({
+      where: { organizationId }
+    });
+
+    if (!organizationId) {
```

Comment:
move this check above the allFaqs. and also don't do !organizationId. First find the organization and check if it exists like its done in other endpoints